Content.Client/Guidebook/Controls/GuideReagentEmbed.xaml.cs

```diff
@@ -128,8 +128,16 @@ private void GenerateControl(ReagentPrototype reagent)
                     continue;
 
                 var groupLabel = new RichTextLabel();
-                groupLabel.SetMarkup(Loc.GetString("guidebook-reagent-effects-metabolism-group-rate",
-                    ("group", _prototype.Index<MetabolismGroupPrototype>(group).LocalizedName), ("rate", effect.MetabolismRate)));
+                if (effect.MetabolismRate >= 0)
+                {
+                    groupLabel.SetMarkup(Loc.GetString("guidebook-reagent-effects-metabolism-group-rate",
+                        ("group", _prototype.Index<MetabolismGroupPrototype>(group).LocalizedName), ("rate", effect.MetabolismRate)));
+                }
+                else
+                {
+                    groupLabel.SetMarkup(Loc.GetString("guidebook-reagent-effects-metabolism-group-norate",
+                        ("group", _prototype.Index<MetabolismGroupPrototype>(group).LocalizedName)));
+                }
```

Comment:
This is bogus/incorrect. Plant metabolism has a rate too and it should be specified, there should *never* be metabolism groups without a rate.

Aside from that, this shouldn't be using the metabolism system. This should be a new dropdown for plant effects, instead of being in the same classification as true metabolizers. In the future this might change if plants are moved to use metabolisms like they should, but for now they shouldn't be merged (imo).

Reply:
The rate is something like 1 unit every 3 seconds and is technically defied in the plant container system and could theoretically depend on the tray used. Too messy to just describe as a flat metabolism and I didn't think hardcoding an explanation specifically for plant metabolism would be a good idea.

And yeah, it's a hack. Thought just slapping it into the whole metabolism thing would be better (less copy-pasting code) than basically running the same code that metabolisms use again.

Reply:
Do what some other things (like thirst) do, use that base tray rate as the 1.0x (so, show it in the UI), and fancy trays can claim to be 2.0x or 0.5x plant metabolism speed.

Reply:
Yeah, once I get it as a separate dropdown listing, adding the unconventional rate should be quite simple.